### PR TITLE
adding feature accessor

### PR DESCRIPTION
**Pull request by mkwu**

### CODE REVIEW

File: script/test [hunk 1]
```diff
@@ -21,7 +21,7 @@ if [ "$FMT_ERRS" -gt "0" ]; then
 fi
 echo "ok  gofmt"
 
-VET_ERRS=$(go tool vet -test $pkgs 2>&1 | tee /dev/tty | wc -l)
```
Comment (promiseofcake) [inline script/test]: The true switch here should be `-all -shadow`
https://github.com/vsco/bricks/pull/75

**Comment by negator** [comment]
We should make this a Bazel repo

**Comment by mkwu** [comment]
@negator yeah agree - but maybe that's next step after we get this changeset in?

**Comment by promiseofcake** [comment]
> We should make this a Bazel repo

Would it be possible to keep glide around too without too much contention on pinned hashes? This is a public repo so it would be best to maintain the current status of it if possible.

**Comment by negator** [comment]
@promiseofcake, we sould generate the WORKSPACE file off of glide.lock, so it should be exactly the same

**Comment by promiseofcake** [comment]
For some reason we are getting failing tests on master: https://travis-ci.org/vsco/dcdr, going to restart the build to see if we have a flaky test.

**Pull request merged**